wrapanapi/entities/vm.py

```diff
@@ -483,14 +483,6 @@ def __init__(self, *args, **kwargs):
                     )
                 )
 
-    @abstractproperty
```

Comment:
Discussed offline - these were causing a flake8 failure due to the class attribute defintions at L464 and L466 above.

Those class attribute definitions are not necessary when using abstractproperty decorator, and I would get rid of them, not these abstract decorated method definitions.


Optionally, `@abstractproperty` was deprecated in 3.3 and can be exchanged for using the `@property` and `@abstractmethod` decorators together.